service/controller/cluster.go

```diff
@@ -538,24 +537,9 @@ func newClusterResources(config ClusterConfig) ([]resource.Interface, error) {
 		}
 	}
 
-	var workerCountResource resource.Interface
-	{
-		c := workercount.Config{
-			Logger: config.Logger,
-
-			ToClusterFunc: toClusterFunc,
-		}
-
-		workerCountResource, err = workercount.New(c)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	}
-
 	resources := []resource.Interface{
 		// Following resources manage controller context information.
 		tenantClientsResource,
-		workerCountResource,
```

Comment:
Does this removal mean the worker count was not used for anything here?

Reply:
It wasn't used, at least I didn't seen it